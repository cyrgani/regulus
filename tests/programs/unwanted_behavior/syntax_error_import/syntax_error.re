# intentional syntax error for testing `syntax_error_import.re`
def(a, x, x)
def(b, y, _())