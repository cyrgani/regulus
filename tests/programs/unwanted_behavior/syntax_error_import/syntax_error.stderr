
programs/unwanted_behavior/syntax_error_import/syntax_error.re:3:1: SyntaxError: missing comma in argument list